src/javascripts/Views/addMastersModal.js

```diff
@@ -0,0 +1,31 @@
+ngapp.controller('addMastersModalController', function($scope, $timeout, errorService, modalService) {
+    let opts = $scope.modalOptions;
+    let selectedFile = opts.handle;
+    let selectedFileIndex = xelib.GetFileLoadOrder(selectedFile);
```

Comment:
indexes and load orders are actually different in xEdit (due to `Skyrim.exe`/`Skyrim.hardcoded.dat` being loaded at load order 0.  this variable name should be `fileLoadOrder` or `selectedFileLoadOrder`, or something similar.